channeld/channel_wire.csv

```diff
@@ -23,6 +23,8 @@ channel_init,,old_remote_per_commit,struct pubkey
 channel_init,,funder,enum side
 channel_init,,fee_base,u32
 channel_init,,fee_proportional,u32
+channel_init,,fee_base_global,u32
+channel_init,,fee_proportional_global,u32
 channel_init,,local_msatoshi,struct amount_msat
```

Comment:
I don't think this is channeld's business: it just needs to know fee_base and fee_proportional.  Not sure this commit is needed at all?

Reply:
thanks, I found a better way